plugins/scaffolder-backend-module-github/src/actions/inputProperties.ts

```diff
@@ -277,6 +277,28 @@ const secrets = {
   type: 'object',
 };
 
+const oidcCustomization = {
+  title: 'Repository Secrets',
```

Comment:
Should we update the titles and description for this too?

Reply:
Yes, updated 🤕